### DISCUSSION TITLE
Cant add new fields and new collection type. showing this message !

### DISCUSSION THREAD

**Discussion by shrutisahatpure** [strapi/strapi#6899] [Legacy Topics]
![Untitleds](https://user-images.githubusercontent.com/50610929/86437475-40af7600-bd22-11ea-9e52-02a8a0de5500.png)

**Comment by shrutisahatpure** [comment]
and when i used strapi develop it was showing '"strapi' is not recognized as an internal or external command,
operable program or batch file."

**Comment by MattieBelt** [comment]
Hi @shrutisahatpure, `npm run develop` or `yarn develop` should do the trick. [NPM is then running](https://docs.npmjs.com/cli/run-script) the defined `"develop"` script in your `package.json`. You can't use `strapi develop` if you haven't installed it globally and added the executables to your PATH.
